addons/mrp/mrp.py

```diff
@@ -852,7 +860,7 @@ def _calculate_qty(self, cr, uid, production, product_qty=0.0, context=None):
         production_qty = uom_obj._compute_qty(cr, uid, production.product_uom.id, production.product_qty, production.product_id.uom_id.id)
 
         scheduled_qty = {}
-        for scheduled in production.product_lines:
+        for scheduled in production.move_lines:
```

Comment:
@rim-odoo 
manually added stock move in "Products to Consume" field is not considered/added in "Produce" wizard, only the bom components are added on wizard("To Consume" field), check this video: http://youtu.be/XHcRaYinjJ0